suricata-ocsf/package.yaml

```diff
@@ -236,7 +279,7 @@ pipelines:
       subscribe "suricata"
       where @name == "suricata.smb"
       this = { event: this }
-      python "self.tree_uid = str(self.event.smb.tree_id)"
+      this.tree_uid = str(event.smb.tree_id)
```

Comment:
```suggestion
      tree_uid = str(event.smb.tree_id)
```

Reply:
But is this then I see, this is already assigned in `this` in line 367.

Reply:
`this` is a keyword and refers to the top-level record. In `set` assignments, you can omit it.